packages/plugins/plugin-deck/src/DeckPlugin.tsx

```diff
@@ -432,16 +426,14 @@ export const DeckPlugin = ({
                   return;
                 }
 
-                const newPlankPositioning = settings.values.newPlankPositioning;
-
                 const processLayoutEntry = (partName: string, entryString: string, currentLayout: any) => {
+                  // TODO(burdon): Option to toggle?
+                  const toggle = false;
```

Comment:
Did you see that you can pass `noToggle` in the intent data?

Reply:
i didn't. thanks. let's discuss how to ux this.